pwa/src/components/handlers/handlerForm.tsx

```diff
@@ -225,19 +225,19 @@ export default function HandlerForm({id, endpointId}) {
                         <div className="row">
                           <div className="col-6">
                               <GenericInputComponent type={"text"} name={"name"} id={"nameInput"} data={handler && handler.name && handler.name}
-                                                     nameOverride={"Name"}/>
+                                                     nameOverride={"Name"} required={true}/>
```

Comment:
Wanneer je een `boolean` prop meestuurt in React én de value `true` is, dan hoef je dat niet expliciet aan te geven. Dat komt dan uit op de volgende syntax:

`... required />` in plaats van `... required={true} />`.

Reply:
Note: als de value `false` zou zijn (en die expliciet meegestuurd zou moeten worden naar een component) dan zou je wél `... required={false} />` moeten schrijven.